.github/workflows/lint-and-test.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       - name: Run cargo test (proof primitives - Dory) (std feature only - i.e. not using blitzar)
         run: cargo test proof_primitive::dory::dory_compute_commitments_test --no-default-features --features="std"
       - name: Run hello_world example
-        run: cargo run --example hello_world --features="blitzar test"
+        run: cargo run --example hello_world --no-default-features --features="test"
       - name: Run posql_db example
         run: bash crates/proof-of-sql/examples/posql_db/run_example.sh
```

Comment:
See other comments. Can we make sure we run this example both with and without `blitzar`?

Reply:
needed to add a run with blitzar and another without one right ?